dts/arm/st/wl/stm32wl.dtsi

```diff
@@ -382,6 +382,15 @@
 			};
 		};
 
+		aes: aes@58001800 {
+			compatible = "st,stm32-aes";
```

Comment:
We miss a binding for this compatible

Reply:
Indeed, added it now as the first commit.